tests/playlist/test_playlists.py

```
@@ -0,0 +1,88 @@
+
+@pytest.mark.asyncio
+async def test_validation_rejects_short_duration(tmp_path):
+    core = DummyCore(str(tmp_path))
+    manager = PlaylistManager(core)
+
+    bad = {
+        "id": "bad",
+        "name": "Bad",
+        "items": [{"scene_id": "s1", "duration_ms": 100}],
+    }
+    with pytest.raises(Exception):
```

Ruff (0.13.1)

Do not assert blind exception: `Exception`

(B017)